### DISCUSSION TITLE
show texture in AR android

### DISCUSSION THREAD

**Discussion by zjafarpishe** [google/model-viewer#4472] [Q&A]
When I change the texture, it shows on the 3d view but does not show in AR

**Comment by zjafarpishe** [comment]
ar-modes="scene-viewer webxr"

**Comment by danilokrulj** [comment]
Use only WebXR, scene-viewer defaults it back to the textures from the gltf file.
So use ar-modes="webxr scene-viewer quick-look"
Or ar-modes="webxr quick-look"

**Comment by zjafarpishe** [comment]
In ar-modes="webxr scene-viewer quick-look" size of the models are not real and it is difficult to recognize the surface

**Comment by elalish** [comment]
@zjafarpishe Thanks for the feedback on our WebXR UX vs scene-viewer's. Since we actually control the WebXR UX, would you mind putting some specific feedback (screen-recordings comparing the two would be very helpful) so we can improve?
